tests/unit/calendar/events.js

```diff
@@ -0,0 +1,104 @@
+define( [
+	"jquery",
+	"./helper",
+	"ui/widgets/calendar"
+], function( $, testHelper ) {
+
+var element;
+
+module( "calendar: events", {
+	setup: function() {
+		element = $( "#calendar" ).calendar();
+	}
+} );
+
+test( "change", function() {
+	expect( 6 );
```

Comment:
Using `assert` seems already possible, see: https://github.com/jquery/jquery-ui/pull/1755/commits/53f39f5008a46a332e3a5416875bcd683eb9a1eb#diff-4b5efcb2dfc23f659687625a1565c9b3R140

I guess we want to use this anywhere?